internal/http/services/owncloud/ocdav/propfind/propfind.go

```diff
@@ -391,13 +392,16 @@ func (p *Handler) getResourceInfos(ctx context.Context, w http.ResponseWriter, r
 
 	// we need to stat all spaces to aggregate the root etag, mtime and size
 	// TODO cache per space (hah, no longer per user + per space!)
-	var rootInfo *provider.ResourceInfo
-	var mostRecentChildInfo *provider.ResourceInfo
-	var aggregatedChildSize uint64
-	spaceInfos := make([]*provider.ResourceInfo, 0, len(spaces))
-	spaceMap := map[*provider.ResourceInfo]spaceData{}
+	var (
+		rootInfo            *provider.ResourceInfo
+		mostRecentChildInfo *provider.ResourceInfo
+		aggregatedChildSize uint64
+		spaceMap            = make(map[*provider.ResourceInfo]spaceData, len(spaces))
+	)
 	for _, space := range spaces {
-		if space.Opaque == nil || space.Opaque.Map == nil || space.Opaque.Map["path"] == nil || space.Opaque.Map["path"].Decoder != "plain" {
+		if space.Opaque == nil ||
```

Comment:
we should probably intdroduce a `func hasMountPath(space)` to improve readability?